backend/app/routes/Q&A/answers/index.js

```diff
@@ -12,7 +12,7 @@ const updateAnswerStatus = require('./updateAnswerStatus');
 router.post('/', validation(answerValidationSchema), postAnswer);
 
 // GET API FOR ANSWERS
-router.get('/:questionId', validation(getAnswerValidationSchema), getAnswers);
+router.get('/:questionId', getAnswers);
```

Comment:
Validation is removed because in validation it is checking for req.body. if not req.boy it throughs error. so removed. Now it works fine. @Kajol-Kumari  If check in the predeploy then you can see only questions not answers. with merging this it will solve.

Reply:
Instead of request.body, change the validation logic to validate req.query na as now u will be reading from query param

Reply:
@Kajol-Kumari it has been changed but in the validation file it is taking request.body only which are applied for post request. but here for get they used that same validation. you can check that other get requests doesn't have that validation.

Reply:
@Kajol-Kumari  that validation check that data comes from body or not. If it's not from body it throughs error. for get we need to use params from url so we doesn't pass any data from body. so that endpoint won't work if that validation is still exist. we already changed get answer by id to request from url. I hope you understand.

Reply:
![kajolcheck](https://github.com/HITK-TECH-Community/Community-Website/assets/106808387/0d58cb31-dd17-422a-a60d-8bd895cb2941)

@Kajol-Kumari  This is the file used there for validation. it is applied for all post requests. If we keep this validation. we won't get any data from get. or if we want to change this validation file. this will impact on other post request, why because this validations are designed for post only. by mistake they added it in the get request. so i removed it. I hope you get my point.